src/main/java/nl/cherement/jak/controller/BoardController.java

```diff
@@ -45,6 +46,13 @@ public HttpStatus deleteById(@PathVariable("id") Long id) {
     public boolean cardMoved() {
         return true;
     }
+
+    @PostMapping("/user/{id}")
+    public HttpStatus addUser(@PathVariable("id") Long boardId, @RequestBody UserEntity user) {
+        service.addUser(boardId, user.getId());
```

Comment:
Je kan een userEntity niet in een RequestBody zetten. Daar zou je UserDTO voor moeten gebruiken. Maar tbh lijkt het me in dit geval beter als je van de UserId een pathVariable maakt. Das toch de enige die je nodig hebt.

Reply:
UserEntity werkt wel maar pathVariable is idd beter.

Reply:
Het werkt wel maar is een security Vulnerability. Daarom kunnen we het niet gebruiken.